specs/collections/spec.yml

```
@@ -0,0 +1,102 @@
+openapi: 3.0.2
+info:
+  title: Collections API
+  description: |
+    The Algolia Collections API lets you create and curate product listings.
+
+    ## Client libraries
+
+    Use Algolia's API clients and libraries to reliably integrate Algolia's APIs with your apps.
+
+    See: [Algolia's ecosystem](https://www.algolia.com/doc/guides/getting-started/how-algolia-works/in-depth/ecosystem/)
+
+    ## Base URLs
+
+    The base URL for requests to the Collections API is:
+
+    - `https://experiences.algolia.com`
```

maybe 
```suggestion
    - `https://experiences.algolia.com/1/collections`
```

as in theory we use this endpoint for other things too